### DISCUSSION TITLE
Can I include the latest version in the URL of docs?

### DISCUSSION THREAD

**Discussion by anthony-df-nguyen** [facebook/docusaurus#8907] [Q&A]
Hello, I looked through the docusaurus documentation and Im having trouble figuring out the solution to the following scenario.
I want to be able to include the version in the URL that the **latest** version can be accessed.

I currently have my docs site set up where:

/docs houses the 'current' docs which is accessible at  mysite.com/next/my-page/ for unreleased documentation.

I also have a latest version 1.0 which is accessible at mysite.com/my-page/
If possible, how would I get the latest version  1.0 to be served at mysite.com/1.0/my-page/ instead?

**Comment by slorber** [comment]
You can customize versioning behavior and path segments on a per-version basis through the plugin options.

Documented here: https://docusaurus.io/docs/versioning#configuring-versioning-behavior
